package.json

```diff
@@ -3,6 +3,16 @@
   "version": "1.4.1",
   "description": "MagicMirror² Module that displays the current moon phase",
   "main": "MMM-MoonPhase.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/NolanKingdon/MMM-MoonPhase.git"
+  },
+  "keywords": [
+    "MagicMirror",
+    "Moon",
+    "Phase",
+    "Astrony"
```

Comment:
Astronomy :slightly_smiling_face:

Reply:
I'll blame it on a slow keyboard and not a slow human.